### DISCUSSION TITLE
Trimming

### DISCUSSION THREAD

**Discussion by anpoit00** [Macad3D/Macad3D#42] [Ideas]
Thank you for a great job!

I have a small question, do you consider about trimmung in sketcher (Modifying sketcher objects by trimming or extending edges)?

**Comment by dipts** [comment]
Can you give an example of how you envision the workflow? 
In the Sketcher, segments can at least be cut - an additional point inserted - and the unwanted segment can then be deleted. Is that close to trimming as you imagine it?

**Comment by anpoit00** [comment]
Thanks for the quick reply. I have found out how to trim with "Weld" option. Have I perhaps overlooked Chamber Option?
I have included a small video on how Catia does it.
https://www.youtube.com/watch?v=GaoEh3uOR3s

**Reply by dipts** [reply]
You are right, a chamfer tool is still missing in the sketcher.

**Reply by anpoit00** [reply]
Do you have plans to implement champfer?
Something else: leg arcs it would be very helpful if the center is also displayed. Then you can only lock the center, for example.

**Reply by dipts** [reply]
> Do you have plans to implement champfer?

Yes, probably, the feature sounds useful. But the list is long, I can't say when it will come in.
